template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/CmTemplateProcessor.java

```diff
@@ -446,7 +450,10 @@ private void addCmVersionDependantConfigs(ClouderaManagerRepo cmRepoDetails, Tem
         }
         if (Objects.nonNull(cmRepoDetails) && CMRepositoryVersionUtil.isEnableKerberosSupportedViaBlueprint(cmRepoDetails)
                 && templatePreparationObject.getKerberosConfig().isPresent()) {
-            instantiator.setEnableKerberos(new ApiConfigureForKerberosArguments());
+            instantiator.setEnableKerberos(new ApiConfigureForKerberosArguments()
+                    .datanodeTransceiverPort(SECURE_DATANODE_ADDRESS)
```

Comment:
do we need to check any cm version?

Reply:
No, this exists since the very first version. I tested with 7.1.0